mmv1/products/workbench/Instance.yaml

```diff
@@ -0,0 +1,471 @@
+  - !ruby/object:Api::Type::NestedObject
+    name: gceSetup
+    default_from_api: true
+    description: The definition of how to configure a VM instance outside of Resources and Identity.
+    properties:
+      - !ruby/object:Api::Type::String
+        name: machineType
+        default_from_api: true
+        description: |
+          Optional. The machine type of the VM instance. https://cloud.google.com/compute/docs/machine-resource
+        pattern: projects/{{project}}/zones/{{location}}/machineTypes/{{name}}
+        diff_suppress_func: 'tpgresource.CompareSelfLinkOrResourceName'
+        custom_flatten: templates/terraform/custom_flatten/name_from_self_link.erb
+      - !ruby/object:Api::Type::Array
+        name: acceleratorConfigs
+        description: |
+          The hardware accelerators used on this instance. If you use accelerators, make sure that your configuration has
+          [enough vCPUs and memory to support the `machine_type` you have selected](https://cloud.google.com/compute/docs/gpus/#gpus-list).
+          Currently supports only one accelerator configuration.
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::Enum
+              name: type
+              values:
+                - ACCELERATOR_TYPE_UNSPECIFIED
+                - NVIDIA_TESLA_P100
+                - NVIDIA_TESLA_V100
+                - NVIDIA_TESLA_P4
+                - NVIDIA_TESLA_T4
+                - NVIDIA_TESLA_A100
+                - NVIDIA_A100_80GB
+                - NVIDIA_L4
+                - NVIDIA_TESLA_T4_VWS
+                - NVIDIA_TESLA_P100_VWS
+                - NVIDIA_TESLA_P4_VWS
+              description: Optional. Type of this accelerator.
+            - !ruby/object:Api::Type::String
+              name: coreCount
+              description: Optional. Count of cores of this accelerator.
+          description: |
+            Optional. The hardware accelerators used on this instance. If you
+            use accelerators, make sure that your configuration has [enough vCPUs and memory
+            to support the `machine_type` you have selected](https://cloud.google.com/compute/docs/gpus/#gpus-list).
+            Currently supports only one accelerator configuration.
+      - !ruby/object:Api::Type::Array
+        name: serviceAccounts
+        description: |
+          The service account that serves as an identity for the VM instance. Currently supports only one service account.
+        default_from_api: true
+        immutable: true
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::String
+              name: email
+              default_from_api: true
+              description: Optional. Email address of the service account.
+            - !ruby/object:Api::Type::Array
+              name: scopes
+              item_type: Api::Type::String
+              default_from_api: true
```

Comment:
This can't be both default_from_api and output. Remove default_from_api probably